src/sidebar/components/ModerationBanner.js

```diff
@@ -60,40 +61,58 @@ function ModerationBanner({ annotation, api, toastMessenger }) {
       });
   };
 
-  const toggleButtonProps = (() => {
-    const buttonProps = {};
-    if (annotation.hidden) {
-      buttonProps.onClick = unhideAnnotation;
-      buttonProps.title = 'Make this annotation visible to everyone';
-    } else {
-      buttonProps.onClick = hideAnnotation;
-      buttonProps.title = 'Hide this annotation from non-moderators';
-    }
-    buttonProps['aria-label'] = buttonProps.title;
-    return buttonProps;
-  })();
-
-  const bannerClasses = classnames('ModerationBanner', {
-    'is-flagged': flagCount !== null && flagCount > 0,
-    'is-hidden': annotation.hidden,
-    'is-reply': annotationMetadata.isReply(annotation),
-  });
-
   if (!isHiddenOrFlagged) {
     return null;
   }
   return (
-    <div className={bannerClasses}>
-      {!!flagCount && !annotation.hidden && (
-        <span>Flagged for review x{flagCount}</span>
-      )}
-      {annotation.hidden && (
-        <span>Hidden from users. Flagged x{flagCount}</span>
+    <div
+      className={classnames(
+        'flex gap-x-3 bg-grey-1 text-color-text font-semibold',
+        // Vertical margins should ultimately be handled by the parent, but
+        // until we can refactor outer components (e.g. `ThreadCard`), this
+        // component manages its own bottom margin
+        'mb-2 ',
+        {
+          // For top-level annotations, use negative margins to "break out" of
+          // the parent card's 3-unit padding and have the banner span the
+          // full card width with no padding
+          '-mt-3 -ml-3 -mr-3': !annotationMetadata.isReply(annotation),
```

Comment:
I should point out that having to break out of prison with negative margins should be no one's idea of a clever solution here. The entire thingy needs to be re-thought at some point, but we've backed ourselves slightly into a corner by having a component that needs to be rendered both at a top-card-level and within replies. I didn't want to completely re-invent the wheel at this time.

At least we've greatly simplified the styling.